packages/blade/src/components/Table/TableBody.web.tsx

```diff
@@ -152,7 +147,7 @@ const TableBody = assignWithoutSideEffects(_TableBody, {
   componentId: ComponentIds.TableBody,
 });
 
-const StyledCell = styled(Cell)<{
+export const StyledCell = styled(Cell)<{
```

Comment:
This will expose StyledCell and CellWrapper to our consumers too since we are exporting from here and also doing `export * from './TableBody' in index.ts